addon/styles/partials/_uni-carrousel.scss

```diff
@@ -0,0 +1,36 @@
+.uni-carrousel {
+    @include carroussel();
```

Comment:
If you could just change the mixin name to ```carrousel``` to keep the consistency, would be nice :)